omnigibson/robots/robot_base.py

```diff
@@ -391,7 +392,7 @@ def visualize_sensors(self):
         frames = dict()
         remaining_obs_modalities = deepcopy(self.obs_modalities)
         for sensor in self.sensors.values():
-            obs = sensor.get_obs()
+            obs = sensor.get_obs()[0]
```

Comment:
nit: `obs, _` for consistency (so it's clear that there's exactly two return values)